docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,14 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* Salespeople of companies
+* has a need to manage a significant number of contacts into either Leads or Contacts
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: manage contacts faster and more efficiently than a typical mouse/GUI driven app
```

Comment:
don't think this is needed

Reply:
Yeah, faster and more efficient are basically synonyms.